lib/src/model/category.dart

```diff
@@ -84,7 +84,7 @@ class Category
   String get name => _categoryDefinition.displayName;
```

Comment:
Rather than introducing the term `_internalName`, perhaps it would be better we changed this to return `_name`.

And then added a `displayName` property.

Obviously, we'd have to refactor the templates to then use `displayName` where they currently use `name`.

Ofcourse, that might not be reasonable, if `name` is used in inherented templates?

Reply:
It is actually called 'internal name' in the comments already, but I actually think this renaming does not belong in this PR, so I reverted it for now.